tuning/config/acceleration_configs/fast_moe.py

```diff
@@ -42,8 +44,15 @@
 @parsable_dataclass
 @dataclass
 class FastMoe:
+    ep_degree: Union[int, bool] = 1
+    disable_distributed: bool = field(
```

Comment:
make it internal variable with `_disable_distributed` if its not to be exposed to the user. 

If it is please make sure to pass the help.

Reply:
We cannot since, we use that in fms-accelerate plugin directly, we need to match that interface, so we have to keep it intact with variable names